controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -798,6 +798,17 @@ public CompletableFuture<CreateStreamStatus.Status> createStream(String scope, S
                     () -> createStreamBody(scope, stream, streamConfig, createTimestamp, context));
     }
 
+    private boolean validateStreamConfig(String stream, StreamConfiguration config) {
```

Comment:
In what sense is it "validating" the config? The name makes it sound like "false" means the config is invalid and an error should be thrown.